usr/share/rear/layout/save/FDRUPSTREAM/Linux-s390/990_copy_disklayout_file.sh

```diff
@@ -0,0 +1,22 @@
+#
+# For s390 if ZVM naming is setup, then copy the disklayout.conf to the output location
+# s390 optional naming override of dsklayout.conf to match the s390 filesytem naming conventions
```

Comment:
The 'copy the disklayout.conf to the output location' functionality
happens in a FDRUPSTREAM specific script
usr/share/rear/layout/save/FDRUPSTREAM/Linux-s390/990_copy_disklayout_file.sh
so that it is only done in case of BACKUP=FDRUPSTREAM.
Is that intended or should that be done for all BACKUP methods?

Reply:
this is intended.  the copy of the disk_layout is not needed for the restore on s390 to work properly.   I was asked to make this file available for FDR

Reply:
I added an explanatory comment to
layout/save/FDRUPSTREAM/Linux-s390/990_copy_disklayout_file.sh
https://github.com/rear/rear/pull/2142/commits/2b8debdb641a4ec184866f0e9d1577235c6712f6